app/components/UI/Notification/Row/Content.tsx

```diff
@@ -0,0 +1,72 @@
+import React, { useCallback } from 'react';
+import { View, StyleSheet } from 'react-native';
+import Text, {
+  TextColor,
+  TextVariant,
+} from '../../../../component-library/components/Texts/Text';
+
+interface NotificationContentProps {
+  title: string;
+  description?: {
+    asset?: {
+      symbol?: string;
+      name?: string;
+    };
+    text?: string;
+  };
+  createdAt: string;
+  value: string;
+  styles: StyleSheet.NamedStyles<any>;
+}
+
+function NotificationContent({
+  title,
+  description,
+  createdAt,
+  value,
+  styles,
+}: NotificationContentProps) {
+  const useRenderTwoPartsTitles = useCallback(() => {
+    const parts = title?.toLowerCase().includes('sent')
+      ? title?.split(/(.*?to)/g)
+      : title?.toLowerCase().includes('received')
+      ? title?.split(/(.*?from)/g)
+      : [];
```

Comment:
You could safeguard this part by first testing title is defined and then rely on this without the redundant optional chaining.
Returning directly in case of undefined or null title.
Also, the ternary here is not the most readable structure. Try a classic if/else, it sometimes makes it more readable with the exact same efficiency.

Finally, be careful, your regex are vulnerable to redos attack (as indficated by Sonar). This may be changed to more simple regex without backtracking or direct substring manipulation.

Example suggestion for all these changes:
```suggestion
if (!title) return <Text color={TextColor.Alternative} variant={TextVariant.BodySM}>{title}</Text>;

    const lowerTitle = title.toLowerCase();
    let parts;

    if (lowerTitle.includes('sent')) {
      parts = title.split(/(\bto\b)/g);
    } else if (lowerTitle.includes('received')) {
      parts = title.split(/(\bfrom\b)/g);
    } else {
      parts = [];
    }
```

Reply:
Good approach, I implemented a mix of it.